hasura/migrations/default/1646252376870_create_table_public_claims/up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE "public"."claims" ("id" bigserial NOT NULL, "distribution_id" bigint NOT NULL, "index" bigint NOT NULL, "proof" varchar NOT NULL, "flag" boolean NOT NULL, "amount" bigint NOT NULL, "user_id" bigint NOT NULL, PRIMARY KEY ("id") , FOREIGN KEY ("distribution_id") REFERENCES "public"."distributions"("id") ON UPDATE no action ON DELETE no action, FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON UPDATE no action ON DELETE no action, UNIQUE ("id"));
```

Comment:
did you want to add the `address` column back in here?

Reply:
Oh my, yes I did, I can add that on my end.

Reply:
Done